client/lib/importer/test/api-interaction.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import Dispatcher from 'dispatcher';
+import nock from 'nock';
+import partial from 'lodash/function/partial';
+
+import { fetchState } from '../actions';
+import { actionTypes } from '../constants';
+import store from '../store';
+
+const fetchTestState = partial( fetchState, 0 );
+const hydratedState = () => store.get().getIn( [ 'api', 'isHydrated' ] );
+const resetStore = () => Dispatcher.handleViewAction( { type: actionTypes.RESET_STORE } );
+
+const queuePayload = payload =>
+	nock( 'https://public-api.wordpress.com:443' )
+		.get( '/rest/v1.1/sites/0/imports/' )
+		.replyWithFile( 200, `${ __dirname }/api-payloads/${ payload }.json` );
+
+describe( 'Importer store', () => {
+	beforeEach( resetStore );
+
+	describe( 'API integration', () => {
+		it( 'should hydrate if the API returns a blank body', done => {
+			expect( hydratedState(), 'before fetch' ).to.be.false;
```

Comment:
yep. it's there in all of the [*chai* assertions](http://chaijs.com/guide/styles/)

the reason I chose to use it here is because I had the same assertion twice in the function and when it failed without the messages, it wasn't clear which one had failed